public/app/plugins/datasource/testdata/QueryEditor.test.tsx

```diff
@@ -37,7 +37,7 @@ describe('Test Datasource Query Editor', () => {
   it('should switch scenario and display its default values', async () => {
     const { rerender } = setup();
 
-    let select = (await screen.findByText('Scenario')).nextSibling!.firstChild!;
+    let select = (await screen.findByText('Scenario')).nextSibling!.firstChild!.firstChild!;
```

Comment:
```suggestion
    let select = (await screen.findByText('Scenario')).nextSibling!.firstChild!;
```

I'm gonna try to apply this suggestion because this change isn't even there when I try to edit the file through GitHub 😩